src/common.ts

```diff
@@ -19,6 +19,7 @@ import {URL} from 'url';
 
 export class GaxiosError<T = any> extends Error {
   code?: string;
+  type?: string;
```

Comment:
Could we add a header here to come some idea of what the expected contents would be?

Reply:
reading their docs, it looks like they actually advocate using `err.name`:

https://github.com/bitinn/node-fetch#request-cancellation-with-abortsignal